### ISSUE TITLE
public personal  docker creds

### ISSUE THREAD

**Issue by Mostafa-ewida** [0xZe/FS#1]
![image](https://github.com/0xZe/FS/assets/71707534/c1f279ac-2cf9-421a-8906-b939859145f9)
this is your dockerhub creds 
mohammedd.eldeeb@gmail.com
0xze:rbir275@

and these are public 
consider doing a timed and scoped dockehub token and deactivate it after you have done using it

**Comment by 0xZe** [comment]
Hi Mostafa,
Thank you very much for your concern and warning.
I have already changed credentials directly after finishing this project.
Again Thank you very much.
ه

‫في الأربعاء، 6 مارس 2024 في 2:29 ص تمت كتابة ما يلي بواسطة
‪Mostafa-ewida‬‏ ***@***.***‬‏>:‬

**Comment by Mostafa-ewida** [comment]
yeah , I know that , I 've tried it to see if it is still valid 😂
here is me number if you wanna talk more about devops 
sorry it is whatsapp only 

```+20 155 232 5653```

and one more thing close the issue after 
and have a good day
